assets/js/translation-events.js

```diff
@@ -37,6 +37,14 @@
 		 * @param isDraft	  Whether the current event status is a draft or not
 		 */
 		function handleSubmit( eventStatus, isDraft ) {
+			if ( '' === $( '#event-start' ).val() ) {
```

Comment:
Do we need to check this on the backend as well? I'm not sure what the behaviour is [here](https://github.com/WordPress/wporg-gp-translation-events/blob/trunk/wporg-gp-translation-events.php#L142-L143) at the moment, maybe it already throws when the hour is not set, but I'm not sure.

Reply:
We check it in the backend [here](https://github.com/WordPress/wporg-gp-translation-events/blob/ae6049aa8ef5a69728fbba5116a4a602c08825ee/wporg-gp-translation-events.php#L184-L192).